ballerina-tests/tests/records.bal

```diff
@@ -243,3 +243,9 @@ public type Languages record {|
 public type CovidEntry record {|
     string isoCode;
 |};
+
+public type Review record {
+  Product product;
+  int score;
+  string description;
+};
```

Comment:
Shall we use closed record here if possible?

Reply:
Updated